### DISCUSSION TITLE
problem while trying to add another email address as a primary email

### DISCUSSION THREAD

**Discussion by 2O0K** [community/community#49988] [New to GitHub]
how can i change the email address that is linked to my gh account? it keeps showing me this error when i try to change from the Settings -> Emails -> Add new email

Error : Error adding MY_EMAIL_ADDRESS email is already in use  

please help me out, thanks in advance!

**Comment by Megagig** [comment]
I am experiencing the same issue. please ow did you solve this problem?

**Comment by itsjaychaturvedi** [comment]
facing the same issue, can anyone help here. @ghostinhershell
